src/main/resources/org/opentripplanner/apis/gtfs/schema.graphqls

```diff
@@ -669,6 +669,8 @@ type Leg {
   For non-transit legs, null.
   """
   headsign: String
+  "An identifier for the leg, which can be used to re-fetch transit leg information."
```

Comment:
Maybe we should also mention that an invalid reference id is returned for other legs. Also, mention that the id might not be usable forever since the data and ids within data can change over time.